investigations/bedrock2/incrementwait.c.out

```diff
@@ -0,0 +1,42 @@
+     = "#include <stdint.h>
+#include <memory.h>
+
```

Comment:
In general I'd prefer not to include generated files, what are your thoughts?

Reply:
I'd like to include it in this investigation because it will help me track how the output changes. I'm happy to not include generated files if I move this stuff out of `investigations`.